spec/requests/api/v1/timelines/link_spec.rb

```diff
@@ -110,6 +119,11 @@
     end
 
     context 'when the instance allows public preview' do
+      before do
+        Form::AdminSettings.new(timeline_preview_local: true).save
+        Form::AdminSettings.new(timeline_preview_remote: true).save
+      end
+
       context 'with an authorized user' do
```

Comment:
This spec passes with this, but the 'with an anonymous user` spec fails with a 422, and I have no idea why, because `require_auth?` evaluates to `false` — I feel like I'm missing something here?